query/src/servers/http/v1/multipart_format.rs

```diff
@@ -282,13 +282,15 @@ impl Processor for SequentialInputFormatSource {
                 let data = match &mut self.input_decompress {
                     None => data,
                     Some(decompress) => {
```

Comment:
Is there a `decompress_bound` method for each compressor?

Reply:
What's `decompress_bound` for?

Reply:
https://docs.rs/zstd-safe/4.1.1+zstd.1.5.0/zstd_safe/fn.decompress_bound.html

Reply:
> Is there a `decompress_bound` method for each compressor?

As far as I know, `async-compression` doesn't expose this method. Maybe we can hack it if needed.